### PR TITLE
feat: Moto Claim Flow (DomW)

### PR DESCRIPTION
**Pull request by domw30**
[Associated Notion Card](https://www.notion.so/zerotech/dApp-Team-Dash-78b824c0cd0e46c59c19dfe103ae929a?p=73df6ef6699c44319d8990771ec1134d)

-----

## 1. Pull request checklist

- [x] Notion card has been moved to the Code Review column
- [x] Notion card has a link to this PR
- [x] A reviewer has been assigned to the Notion card

## 2. PR type
- Feature

## 3. What is the old behaviour?
Claim flow does not exist
Unable to open claim nft via Wheels NFT View

## 4. What is the new behaviour?
Claim flow added
Able to open claim nft via Wheels NFT View

### CODE REVIEW

File: src/pages/ZNS/ZNS.tsx [hunk 1]
```diff
@@ -260,7 +260,7 @@ const ZNS: React.FC<ZNSProps> = () => {
 				/>
 			)}
 			{/* ZNS Content */}
-			{enableBanner && <Raffle />}
+			{enableBanner && <ClaimNFT requireBanner />}
```
Comment (padala) [inline src/pages/ZNS/ZNS.tsx:263]: @domw30 Given time constraints, I think it is better idea to use existing banner flow as claim flow is similar to mint drop flow and just update the click action to launch claim flow.

File: src/components/MintPreview/MintPreview.tsx [hunk 2]
```diff
@@ -90,29 +100,23 @@ const MintPreview = (props: MintPreviewProps) => {
...
+										</FutureButton>
+									)}
+								</div>
 								<div
 									className={`${styles.TextContainer} ${
 										exists ? styles.Success : ''
 									}`}
 								>
+									<div className={styles.IconContainer}>
+										<img alt={ALT_TEXT.QUESTION_MARK} src={questionMark} />
```
Comment (padala) [inline src/components/MintPreview/MintPreview.tsx:116]: this is being used as part of mint flow too. Are these changes intended for both flows?

Comment (domw30) [reply src/components/MintPreview/MintPreview.tsx:116]: Added a comment to discord moto channel.

https://user-images.githubusercontent.com/39112648/171621098-f832cf9f-ccf2-482e-87ba-f4f051d8fdb3.mp4

File: src/containers/flows/ClaimNFT/index.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+					<ClaimNFT
+						openConnect={openConnect}
+						onClose={closeWizard}
+						onSubmit={onSubmit}
+					/>
+				</Overlay>
+			)}
+			{requireBanner ? (
+				<div className={styles.BannerContainer}>
+					<MintDropNFTBanner
```
[the remark this answers is not in the document]
Comment (domw30) [reply src/containers/flows/ClaimNFT/index.tsx:74]: This condition was required to open claim nft modal from nft view page (option currently visible on all nft view pages).

Did you mean I should replace ```MintDropNFTWizard``` in ```MintDropNFTContainer``` with ```ClaimNFT``` rather than having a ```ClaimNFTContainer```?

**Review by domw30 — COMMENTED** [review]
Note :: Changes here have been merged into feat/zauction-v2.1. PR - https://github.com/zer0-os/zNS-dapp/pull/608
